src/docs/metrics/trading-volume/index.md

```diff
@@ -11,6 +11,10 @@ The Trading Volume is the volume in USD traded on exchanges.
 The metric is taking into consideration all the trading pairs. The trading pairs that
 are not pegged to USD are multiplied by the USD price of the asset.
 
+Trading Volume in USD metrics:
+- `volume_usd` - Trading volume in USD
+- `volume_usd_5m` - Average trading volume in USD over five-minute intervals
```

Comment:
This is not a correct definition, or at least it's misleading.

Our trading volume always represents the 24h sliding window value.
`volume_usd` is the raw data we get.
`volume_usd_5m` is a precomputed intraday metric that aligns the values at 05:00 minutes and fills the gaps with the last known value. 

I think a general overhaul of the definition would be nice to have.

Reply:
Added new description